console/src/metadata/queryUtils.ts

```diff
@@ -629,13 +640,43 @@ export const getConsoleStateQuery = {
   args: {},
 };
 
-export const getInvocationLogsQuery = (
-  eventType: 'one_off' | 'cron',
-  eventID: string
-) => ({
-  type: 'get_event_invocations',
-  args: {
-    type: eventType,
-    event_id: eventID,
-  },
-});
+export type SupportedEvents = 'cron' | 'one_off';
+
+export const getEventInvocationsLogByID = (
+  type: SupportedEvents,
+  event_id: string
+) =>
+  getMetadataQuery(
+    'get_event_invocations',
+    'default',
```

Comment:
we shouldn't pass the source. server will take care of that and will make this query on the proper source